sycl/source/detail/global_handler.cpp

```diff
@@ -47,7 +80,24 @@ T &GlobalHandler::getOrCreate(InstWithLock<T> &IWL, Types... Args) {
   return *IWL.Inst;
 }
 
-Scheduler &GlobalHandler::getScheduler() { return getOrCreate(MScheduler); }
+void GlobalHandler::attachScheduler(Scheduler *Scheduler) {
+  // The method is for testing purposes. Do not protect with lock since
```

Comment:
```suggestion
  // The method is used in unittests only. Do not protect with lock since
```

Reply:
:-) fixed in https://github.com/intel/llvm/pull/6837/commits/71e9048af88ddcc853968cf9bbaa050f63ad323e